src/promptflow/promptflow/_sdk/data/docker/start.sh.jinja2

```diff
@@ -1,4 +1,8 @@
 # Startup script for the flow container
+
+CONDA_ENV_PATH="$(conda info --base)/envs/{{env.conda_env_name}}"
```

Comment:
will we have multiple app/process running in one container? if so, may need to copy this export to each script.

Reply:
It seems we will only have one app  in one container for now.